akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/FilterStage.scala

```diff
@@ -252,6 +252,7 @@ import scala.util.matching.Regex
       private def replay(entityOffset: EntityIdOffset): Unit = {
         val entityId = entityOffset.entityId
         val fromSeqNr = entityOffset.seqNr
+        // FIXME what about the replicaId when using RES?
```

Comment:
Consumer defined filters may just work, but I think we are loosing the replicaId in the translation. The streamId is always the same as the entityType.